lib/github_changelog_generator/generator/entry.rb

```diff
@@ -40,6 +40,15 @@ def generate_entry_for_tag(pull_requests, issues, newer_tag_name, newer_tag_link
       @content
     end
 
+    def line_labels_for(issue)
```

Comment:
Can you move this method where it was, so it's easier to compare with the original version?

Reply:
@ferrarimarco Boss I can not. It is up here because it needs to be in `public` part. If you move it down there, this method will be private and not accessible

Reply:
Right, it's called publicly later, so this can't move. Thanks, @dusan-dragon, for explaining!